lerobot/common/policies/act/modeling_act.py

```diff
@@ -38,7 +38,12 @@
 from lerobot.common.policies.normalize import Normalize, Unnormalize
 
 
-class ACTPolicy(nn.Module, PyTorchModelHubMixin):
+class ACTPolicy(nn.Module, PyTorchModelHubMixin, 
+                library_name="lerobot",
+                repo_url="https://github.com/huggingface/lerobot",
+                docs_url="https://github.com/huggingface/lerobot/",
+                tags=["robotics"],
```

Comment:
```suggestion
                tags=["act-policy", "robotics"],
```

Adding the policy type for each class could help as well => also for discoverability + analytics + so that users know how to reload the models